apps/frontend/src/ui/Dialog/Dialog.tsx

```diff
@@ -31,7 +31,7 @@ export const DialogContent = forwardRef<HTMLDivElement, Props>(
                                 {title}
                             </DialogPrimitive.Title>
                             <DialogPrimitive.Close aria-label="Close">
-                                <XIcon />
+                                <XIcon className="absolute right-[18px] top-[14px] inline-flex" />
```

Comment:
@kaf-lamed-beyt I can see you moved this icon up, I thought it was in line with the design, why was it moved?